scalastyle-config.xml

```diff
@@ -18,59 +18,29 @@
   <parameters>
    <parameter name="regex"><![CDATA[println]]></parameter>
   </parameters>
  </check>
- <!-- <check level="error" class="org.scalastyle.scalariform.NumberOfTypesChecker" enabled="true">
-  <parameters>
-   <parameter name="maxTypes"><![CDATA[30]]></parameter>
-  </parameters>
- </check> -->
- <!-- <check level="error" class="org.scalastyle.scalariform.CyclomaticComplexityChecker" enabled="true">
-  <parameters>
-   <parameter name="maximum"><![CDATA[10]]></parameter>
-  </parameters>
- </check> -->
  <check level="error" class="org.scalastyle.scalariform.UppercaseLChecker" enabled="true"></check>
  <check level="error" class="org.scalastyle.scalariform.SimplifyBooleanExpressionChecker" enabled="true"></check>
- <check customId="method.length" level="error" class="org.scalastyle.scalariform.MethodLengthChecker" enabled="true">
-  <parameters>
-   <parameter name="maxLength"><![CDATA[60]]></parameter>
-  </parameters>
- </check>
- <!-- <check level="error" class="org.scalastyle.file.NewLineAtEofChecker" enabled="true"></check> -->
- <!-- <check level="error" class="org.scalastyle.file.NoNewLineAtEofChecker" enabled="false"></check> -->
  <check customId="method.type" level="error" class="org.scalastyle.scalariform.PublicMethodsHaveTypeChecker" enabled="true">
   <parameters>
    <parameter name="ignoreOverride">true</parameter>
   </parameters>
  </check>
-
   <check level="error" class="org.scalastyle.scalariform.EmptyClassChecker" enabled="true"/>
   <check level="error" class="org.scalastyle.scalariform.ForBraceChecker" enabled="true"/>
-  <!-- <check level="error" class="org.scalastyle.scalariform.ImportGroupingChecker" enabled="true"/> -->
-  <!-- <check level="error" class="org.scalastyle.scalariform.MultipleStringLiteralsChecker" enabled="true"/> -->
   <check level="error" class="org.scalastyle.scalariform.NoCloneChecker" enabled="true"/>
   <check level="error" class="org.scalastyle.scalariform.NoFinalizeChecker" enabled="true"/>
   <check level="error" class="org.scalastyle.scalariform.NoWhitespaceAfterLeftBracketChecker" enabled="true"/>
-  <!-- <check level="error" class="org.scalastyle.scalariform.NoWhitespaceBeforeLeftBracketChecker" enabled="true"/> -->
   <check level="error" class="org.scalastyle.scalariform.NotImplementedErrorUsage" enabled="true"/>
   <check level="error" class="org.scalastyle.scalariform.ProcedureDeclarationChecker" enabled="true"/>
   <check level="error" class="org.scalastyle.scalariform.RedundantIfChecker" enabled="true"/>
-  <!-- <check level="error" class="org.scalastyle.scalariform.StructuralTypeChecker" enabled="true"/> -->
-  <!-- <check level="error" class="org.scalastyle.scalariform.TodoCommentChecker" enabled="true"/> -->
   <check level="error" class="org.scalastyle.scalariform.UppercaseLChecker" enabled="true"/>
-  <check level="error" class="org.scalastyle.scalariform.VarFieldChecker" enabled="true"/>
```

Comment:
I take it this comes up a lot in `scala-js`?

Reply:
It comes often on `scala.js` and there are certain legitimate uses of `var`. I guess this would be good if it could flag `public` uses of `var` but local uses are certainly valid